framework/src/main/java/org/checkerframework/framework/util/typeinference8/types/CaptureVariable.java

```diff
@@ -31,11 +31,13 @@
 
   @Override
   public String toString() {
-    // Use "b" instead of "a" like super so it is apparent that this is a capture variable.
-    if (variableBounds.hasInstantiation()) {
-      return "b" + id + " := " + variableBounds.getInstantiation();
-    }
-    return "b" + id;
+    return String.format("captured %s from %s", typeVariableJava, invocation);
+
+    //    // Use "b" instead of "a" like super so it is apparent that this is a capture variable.
+    //    if (variableBounds.hasInstantiation()) {
+    //      return "b" + id + " := " + variableBounds.getInstantiation();
+    //    }
+    //    return "b" + id;
```

Comment:
Can this be removed?

Reply:
I left it because it is easier to read for debugging.  Eventually I'll probably add a command line argument to switch between the two.